app/lib/theme/widgets/SelectParentWidget.tsx

```diff
@@ -86,12 +86,20 @@ const SelectParentWidget: React.FunctionComponent<
       </Dropdown>
       <style jsx>
         {`
-          :global(input) {
+          div :global(input) {
             width: 100%;
           }
+          div :global(.pg-select-wrapper) {
+            padding: 9px 2px 9px 2px;
+            margin-bottom: 5px;
+            margin-top: 2px;
+          }
```

Comment:
Those should be applied to the `SelectWidget` as well

Reply:
Not sure if you mean `SelectWidget` or `SelectProjectStatusWidget.tsx` or both.  Applied css to both for now.

Reply:
I guess both, but there's probably some refactoring needed there (in another PR). I would expect `SelectProjectStatusWidget` to use `SelectWidget`